diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/CompactInitialization.kt

```diff
@@ -145,7 +147,24 @@ class CompactInitialization(configRules: List<RulesConfig>) : DiktatRule(
         ?: run {
             // add apply block
             property.node.run {
-                val newInitializerNode = kotlinParser.createNode("${property.initializer!!.text}.apply {}")
+                val isNotRequiresParentheses = property.initializer.let {
```

Comment:
May be move this negation into the code below? Thus, we will have 
```kotlin
val isRequireParentheses = property.initializer.let {
!(it is KtParenthesizedExpression || it is KtDotQualifiedExpression ||
                            it is KtCallExpression || it is KtReferenceExpression)
}
````

And further 
```kotlin
if (isRequireParentheses) {
   append("(")
}
```

Reply:
Thanks, changed